content/documentation/staging/tutorial/06-secure/01-traffic-policies.adoc

```diff
@@ -0,0 +1,128 @@
+---
+title: Authorization Policies
+weight: 56
+---
+:icons: font
+:sectlinks:
+
+link:https://istio.io/latest/docs/concepts/security/[Security, window="_blank"] is one of the main pillar of Istio features.
+
+The Istio https://istio.io/latest/docs/concepts/security/#high-level-architecture[Security High Level Architecture] provides a comprehensive solution to design and implement multiple security scenarios.
+
+In the context of this tutorial we are going to show how Kiali can combine telemetry information to create security policies for the workloads deployed on a given namespace.
+
+Istio aggregates in the telemetry the ServiceAccount information used by the workloads in the communication, and this information can be used to define authorization policies with deny and allow actions for a particular live traffic communication status.
```

Comment:
Split into sentences:

Istio telemetry aggregates the ServiceAccount information used in the workloads communication.  This information can be used to define authorization policies that deny and allow actions on future live traffic communication status.

"on future live traffic communication status."  is a little unclear to me but I don't have a suggestion.